cgnet/tests/test_simulation.py

```diff
@@ -589,7 +592,8 @@ def test_saving_all_quantities():
         assert traj.shape[1] == sim_length / save_interval
         file_list = os.listdir(tmp)
 
-        assert len(file_list) == n_expected_files * 4  # coords, forces, pot, ke
+        assert len(file_list) == n_expected_files * \
+            4  # coords, forces, pot, ke
```

Comment:
i would keep this the old way for readability

Reply:
Good catch! Agreed.